src/apps/material/material.test.ts

```diff
@@ -539,6 +539,34 @@ describe("Material", () => {
       .should("have.attr", "aria-pressed", "false");
   });
 
+  it("Can favorite a material", () => {
```

Comment:
yeah like this one you introduced - it does a singular thing. Wonderful 🎉 So if it is failing we know that it's this one very precise thing that's causing trouble.